### PR TITLE
Bumped version to 3.3.3, updated AUTHORS & RELEASE-NOTES

### PR DESCRIPTION
**Pull request by cculianu**
Mark & I are in favor of bumping the version to 3.5:

- This makes it clear that 3.3.x is BCH + BSV and 3.5+ is BCH only.
- This avoids the confusing that exists now with users thinking 3.4 is better/newer/stronger than 3.3. 
- It feels good man.  3.5 has a nice ring to it.

I also went ahead and updated the AUTHORS file.  Let me know if I missed anything in the AUTHORS file or if anything needs changing.

Also added what I remembered happening from the commit logs between last release and now to RELEASE NOTES.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -9,4 +9,13 @@ Genjix - Porting pro-mode functionality to lite-gui and worked on server
...
 rdymac - Website and translations.
-kyuupichan - Miscellaneous.
\ No newline at end of file
+kyuupichan - Miscellaneous.
+Sombernight = many fixes
+Jonald Fyookball - Imperator
+Calin Culiannu - iOS version, MacOS packaging, MacOS QRScanner, miscellaneous
+Malcolm Smith - New Android Version, tuneups and fixups
+Roger Taylor - new version 1.4 blockchain synch code, external plugin system, miscellanous
+Mark Lubdeberg - Windows zbar, Coin Token work, various miscellanity
```
Comment (rt121212121) [inline AUTHORS]: Lund?

Comment (cculianu) [reply AUTHORS]: Lol. Typo. 

Anything else you want to appear next to your name? Can be anything even a joke.

Comment (cculianu) [reply AUTHORS]: I’ll fix when I get home

**Comment by cculianu** [comment]
Ok, I corrected your name. All apologies.. I know how to spell it.  A typo... 😄

**Comment by cculianu** [comment]
UPDATE: Spoke to Jonald.  He doesn't want the version to be 3.5.0 yet... :( 

If you can convince him of otherwise, please do.

Until then, I'm making this PR be 3.3.3

**Comment by cculianu** [comment]
@markblundeberg if it makes you feel better I misspelled my own name!

**Pull request merged**